docs/APIReference-Editor.md

```diff
@@ -144,25 +144,25 @@ or if you just want to display content for a static use case.
 
 Default is `false`.
 
-#### spellCheck
```

Comment:
Was this change supposed to be part of this PR or is it a bad merge ?

Reply:
whoops, will fix